public-ui/src/app/data-browser/charts/chart-base/chart-base.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Injector, Input, } from '@angular/core';
```

Comment:
This is now the base of all of the chart components. It holds all of the globals. I'm pretty sure there can’t be new Highcharts elements such as "barChartOptions" without being a variable in the base component first